pkg/network/ebpf/c/runtime/tracer.c

```diff
@@ -131,39 +131,65 @@ int kprobe__tcp_close(struct pt_regs *ctx) {
     u64 pid_tgid = bpf_get_current_pid_tgid();
     sk = (struct sock *)PT_REGS_PARM1(ctx);
 
-    // Should actually delete something only if the connection never got established
-    bpf_map_delete_elem(&tcp_ongoing_connect_pid, &sk);
```

Comment:
Is this not needed at all? I see we moved `clear_sockfd_maps` to the bottom but this one seems to not be there.